packages/config-storybook/src/main.ts

```diff
@@ -96,6 +96,11 @@ export function configure({
 				...(config.resolve!.plugins ?? []),
 				new ResolveTypescriptPlugin(),
 			]
+			config.resolve!.alias = {
+				...(config.resolve?.alias || {}),
+				'@thematic/react': require.resolve('@thematic/react'),
+				'@fluentui/react': require.resolve('@fluentui/react'),
```

Comment:
do we need styled-components? and how does that interact with @essex/styled-components?

Reply:
Ah, right, styled-components has a context API